packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -342,6 +343,10 @@ export function getPrismaClient(config: GetPrismaClientConfig) {
         validatePrismaClientOptions(optionsArg, config.datasourceNames)
       }
 
+      const logEmitter = new EventEmitter().on('error', () => {
+        // to prevent unhandled error events
```

Comment:
Should we at least log or DEBUG these?

Reply:
Hey @danstarns I was about to a debug trace in here like suggested, but then ended up writing a clarification comment and not doing it. Let me see if it makes sense.

From https://github.com/prisma/prisma/pull/16287/commits/a13c40cf1d4a44c2bc11b9b5039cd4a678b9b432

```typescript
      const logEmitter = new EventEmitter().on('error', (e) => {
        // this is a no-op to prevent unhandled error events
        //
        // If the user enabled error logging this would never be executed. If the user did not
        // enabled error logging, this would be executed, and a trace for the error would be logged
        // in debug mode, which is like going in the opposite direction than what the user wanted by
        // not enabling error logging in the first place.
      })
```